examples/static/README.md

```diff
@@ -0,0 +1,22 @@
+# Serve static files
+
+This example is a modified version of the skeleton code,
+designed to demonstrate serving static files.
+
+This uses Express to handle most of the logic, making
+it very simple to just drop HTML, JS, CSS, and supporting files
+into one or more directories and then serve them, provided they exist.
+
+Note there are additional dependencies in the package.json
+- Express
+- Path
+
+## Caveats
+- The static files will be served at the path declared in the code, so any links in the HTML should be relative links
```

Comment:
I don't think the relative link bit is quite right. They don't have to be relative links (your example uses an absolute link, e.g., `/assets/`). Maybe something like this would be more clear:
```suggestion
- While the files are stored in the `public` subfolder (e.g., `/public/assets/hello.jpg`), this example serves those files at the root of the server. Make sure that your frontend uses the correct path (e.g., `/assets/hello.jpg`).
```